pkg/monitoring/metrics/virt-controller/vmistats_collector.go

```diff
@@ -99,7 +99,7 @@ var (
 				"interface associated with the VMI in the 'address' label, and about the type of address, such as " +
 				"internal IP, in the 'type' label.",
 		},
-		[]string{"node", "namespace", "name", "network_name", "address", "type"},
+		[]string{"node", "namespace", "name", "vnic_name", "interface_name", "address", "type"},
```

Comment:
@machadovilaca should it be interface_name or just interface?

Reply:
interface, I talked with Aviv, and its the same as we did for the vnic and binding names in kubevirt_vm_vnic_info